README.rst

```diff
@@ -225,6 +225,17 @@ conditionally skipped if a dependency is not available.
         >>> import asdf
         >>> asdf.open('file.asdf')
 
+Finally, it is possible to skip collecting doctests in entire subpackages by
+using the ``doctest_subpackage_requires`` option in ``pytest.ini`` or
```

Comment:
What about `tox.ini`? I think that is one of the 3 files that `pytest` will look for; see https://docs.pytest.org/en/latest/customize.html#finding-the-rootdir

Reply:
In fact elsewhere in the README we only mention setup.cfg so I've updated it to match other parts of the README.